backend/models/group.js

```diff
@@ -579,6 +601,21 @@ function clean(groupData) {
 		cleaned.updatedAt = cleaned.updatedAt.getTime();
 	}
 
+	if (cleaned.rules && 0 === cleaned.rules.length) {
+		delete cleaned.rules;
+	}
+
+	for (let i = 0; cleaned.objects && i < cleaned.objects.length; i++) {
+		if (cleaned.objects[i].ifc_guids && 0 === cleaned.objects[i].ifc_guids.length) {
+			delete cleaned.objects[i].ifc_guids;
+		}
+		if (cleaned.objects[i].shared_ids && 0 === cleaned.objects[i].shared_ids.length) {
```

Comment:
Can we add a rule where if both ifc_guids and shared_ids are both empty, remove the entry from the object array?

We need to avoid having entries that have neither. It's ok with have an empty object array, but things go haywire on the frotn end if we have object entries that doesn't have actual entries